gapis/api/cmd_id_group.go

```diff
@@ -221,6 +230,12 @@ func (g CmdIDGroup) Index(index uint64) SpanItem {
 	for _, s := range g.Spans {
 		c := s.itemCount()
 		if index < c {
+			// If the CmdIDGroup is a subcommand group, need to combine the SubCmdIdx
+			// of the immediate parent SubCmdRoot of this CmdIDGroup with the index
+			// target.
+			if x, ok := s.item(index).(SubCmdIdx); ok {
+				return append(g.Parent, x...)
```

Comment:
F2F discussion: This is mixing relative with absolute. Please just be relative (no change).

Reply:
Fixed in #1064